python/ql/lib/semmle/python/ApiGraphs.qll

```diff
@@ -243,6 +243,11 @@ module API {
      */
     Node getAwaited() { result = this.getASuccessor(Label::await()) }
 
+    /**
+     * Gets a node representing a subscript of this node.
```

Comment:
I think an example would be helpful. For `obj[x]`, it's not 100% clear whether you will get just the index `x` or the whole expression.

```suggestion
     * Gets a node representing a subscript of this node.
     * For example `obj[x]` is a subscript of `obj`.
```